datafusion/physical-expr/src/aggregate/hyperloglog.rs

```diff
@@ -115,6 +115,7 @@ where
 
     /// Get the register histogram (each value in register index into
     /// the histogram; u32 is enough because we only have 2**14=16384 registers
+    #[allow(dead_code)]
```

Comment:
After moving to `function-aggregate`, I think we can remove the original file `datafusion/physical-expr/src/aggregate/hyperloglog.rs`.

Reply:
sure